python/cuml/random_projection/rproj.pxi

```diff
@@ -0,0 +1,28 @@
+cdef extern from "random_projection/rproj_c.h" namespace "ML":
```

Comment:
Missing header, 

```suggestion
#
	# Copyright (c) 2019, NVIDIA CORPORATION.
	#
	# Licensed under the Apache License, Version 2.0 (the "License");
	# you may not use this file except in compliance with the License.
	# You may obtain a copy of the License at
	#
	#     http://www.apache.org/licenses/LICENSE-2.0
	#
	# Unless required by applicable law or agreed to in writing, software
	# distributed under the License is distributed on an "AS IS" BASIS,
	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
	# See the License for the specific language governing permissions and
	# limitations under the License.
	#
	
	# cython: profile=False
	# distutils: language = c++
	# cython: embedsignature = True
	# cython: language_level = 3
	
cdef extern from "random_projection/rproj_c.h" namespace "ML":
```
And I don't see why to use a `.pxi` which If I'm not mistaken is an older style of cython and sort of discouraged, with `.pxd` being the norm nowadays.

Reply:
Ok i'll put it in the .pxd file.

Reply:
I put everything inside the .pyx file instead.